crates/gourgeist/src/bare.rs

```diff
@@ -208,6 +217,7 @@ pub fn create_bare_venv(location: &Utf8Path, interpreter: &Interpreter) -> io::R
             interpreter.base_exec_prefix().to_string_lossy().to_string(),
         ),
         ("base-executable", base_python.to_string()),
+        ("prompt", prompt.to_string()),
```

Comment:
We don't really want to add the entry in the `pyvenv.cfg` file is the user pass in  the `--prompt` flag. That's the behavior of `venv` / `virtualenv`. I think we should match that for compatibility.

What you can do here is to add a new parameter to `write_cfg` below which would be an `Option` and then update the function to add the entry if it's `Some`.

Reply:
I changed `write_cfg()` to:

```rust
        if !value.is_empty() {
            writeln!(f, "{key} = {value}")?;
        }
```

"prompt =" is not written when promt is None.